dazel.py

```diff
@@ -133,9 +136,10 @@ def from_config(cls):
                                           DEFAULT_WORKSPACE_HEX))
 
     def send_command(self, args):
+        term = shutil.get_terminal_size()
         command = "%s exec -i -e COLUMNS=%s -e LINES=%s -e TERM=%s %s %s %s %s %s %s %s" % (
             self.docker_command,
-            *shutil.get_terminal_size(),
+            term.columns, term.lines,
```

Comment:
I don't mind too much, just curious, the trick with the * didn't work?

Reply:
Syntax error for me.